src/main/scala/it/bitrock/dvs/api/core/poller/FlightListKafkaPollerCache.scala

```diff
@@ -31,7 +31,10 @@ class FlightListKafkaPollerCache(
 
     case flights: FlightReceivedList =>
       logger.debug(s"Got a $flights from Kafka Consumer")
-      if (flights.elements.nonEmpty) context.become(active(flights))
+      if (flights.elements.nonEmpty) {
+        implicit val reverseUpdatedOrdering: Ordering[FlightReceived] = Ordering.by[FlightReceived, Long](_.updated).reverse
```

Comment:
the scope of the implicit is very narrow, maybe passing it explicitly will be clearer without cluttering the code